README.md

```diff
@@ -30,15 +30,16 @@ jupyter notebook examples/an_awesome_example.ipynb
 
 ## Table of Contents
 
-| Example                                                                                           | Description                                                                                      |
-| ------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------ |
-| [comparing_YOLO_and_EfficientDet](examples/comparing_YOLO_and_EfficientDet.ipynb)                 | Compares the YOLOv4 and EfficientDet object detection models on the COCO dataset                 |
-| [emotion_recognition_presidential_debate](examples/emotion_recognition_presidential_debate.ipynb) | Analyzing the 2020 US Presidential Debates using an emotion recognition model                    |
-| [evaluate_detections](examples/evaluate_detections.ipynb)                                         | Using FiftyOne to quantitatively and qualitatively evaluate object detections                    |
-| [image_uniqueness](examples/image_uniqueness.ipynb)                                               | Using FiftyOne's image uniqueness method to analyze and extract insights from unlabeled datasets |
-| [structured_noise_injection](examples/structured_noise_injection.ipynb)                           | Visually exploring a method for structured noise injection in GANs from CVPR 2020                |
-| [visym_pip_175k](examples/visym_pip_175k.ipynb)                                                   | Exploring the People in Public 175K Dataset from Visym Labs with FiftyOne                        |
-| [wrangling_datasets](examples/wrangling_datasets.ipynb)                                           | Using FiftyOne to load, manipulate, and export datasets in common formats                        |
+| Example                                                                                           | Description                                                                                       |
+| ------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------- |
+| [comparing_YOLO_and_EfficientDet](examples/comparing_YOLO_and_EfficientDet.ipynb)                 | Compares the YOLOv4 and EfficientDet object detection models on the COCO dataset                  |
+| [deepfakes_in_politics](examples/deepfakes_in_politics.ipynb)                                     | Evaluating deepfakes using a deepfake detection algorithm and visualizing the results in FiftyOne |
```

Comment:
FYI- I alphabetized here. Let's keep this ToC alphabetized to make it easier to find examples once there's a lot of them. 

Maybe in the future we'll break out by topics, too